app/open_ai_client.rb

```diff
@@ -28,22 +35,18 @@ def replace_placeholder(text, placeholder, value)
 
   def prompt
     @system_prompt
-    .gsub("${ROLE_PROMPT}", default_role_prompt)
-    .gsub("${INPUT_DESCRIPTION}", default_input_prompt)
-    .gsub("${USER_PROMPT}", default_user_prompt)
-    .gsub("${SUBMISSION}", "#{Submission.new.checklist}")
-    .gsub("${OUTPUT_DESCRIPTION}", default_output_prompt)
+    .gsub("@@SUBMISSION@@", "#{Submission.new.checklist}")
```

Comment:
`${}` is a common syntax we use across the lms for system replacement; Why create a new one `@@`?

Reply:
Will try with `\` slash in front of `$`

Reply:
Which one do think the best approach if `\` works

- Asking user to use \ when using ${} in target github action.
- Replacing the ${} with /${} in workflow before processing and replace the /${} with ${} in code